src/components/pages/ChildCard/ChildCard.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
```

Comment:
This file should be in the 'common' folder since it is a component and not a page.

Reply:
Ok, I made that change and also renamed it to NewChildCard as there was an old one we aren't working with already there